bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -9,8 +9,6 @@
 
 {% block gtm_page_id %}data-gtm-page-id="Homepage"{% endblock %}
 
-{% block page_image %}{{ static('img/home/page-image.png') }}{% endblock %}
```

Comment:
I'm removing this as the image is of the old home page design. It will now fall back to [this image](https://www.mozilla.org/media/img/mozorg/mozilla-256.jpg), which is also out of date as it's the old logo. We should probably file a bug to update this separately.